packages/blade/src/components/Breadcrumb/Breadcrumb.web.tsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react-native-a11y/has-valid-accessibility-role */
+import React from 'react';
+import type { BreadcrumbProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import { Text } from '~components/Typography';
+import { makeAccessible } from '~utils/makeAccessible';
+import { getStyledProps } from '~components/Box/styledProps';
+
+const Separator = ({
+  size,
+  color,
+}: Pick<BreadcrumbProps, 'size' | 'color'>): React.ReactElement => {
+  return (
+    <Text
+      as="span"
+      color={color === 'white' ? 'surface.text.staticWhite.muted' : 'surface.text.gray.muted'}
+      size={size}
+      weight="medium"
+    >
+      /
+    </Text>
+  );
+};
+
+const listStyle = { listStyle: 'none' };
+
+const Breadcrumb = ({
+  size = 'medium',
+  color = 'primary',
+  showLastSeparator = false,
+  accessibilityLabel = 'Breadcrumb',
```

Comment:
we shouldn't have default label no? text will be announced by default I believe

Reply:
We should have default. 

A page should ideally have 1 breadcrumb only, thus it should be announced as "Breadcrumb" - This pattern is set on [APG](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/) too.